tests/boardfarm_plugins/boardfarm_sample_plugin/prplmesh_config.json

```diff
@@ -0,0 +1,49 @@
+{
+    "prplmesh_dummy": {
+        "name": "gateway",
+        "board_type": "prplmesh_docker",
+        "role": "controller",
+        "delay": 7,
+        "conn_cmd": "",
+        "cleanup_cmd": "docker container rm -f gateway",
+        "connection_type": "local_cmd",
```

Comment:
Please limit the contents of this file to what is actually used/required. I believe that would be just cleanup_cmd and connection_type.